linux/meta/collectd.yml

```diff
@@ -1,11 +1,17 @@
+{%- from "linux/map.jinja" import monitoring with context %}
 local_plugin:
   linux_network_netlink:
     plugin: netlink
     template: linux/files/collectd_netlink.conf
-    ignore_selected: false
-    {%- if 'linux_netlink.ls' in salt.keys() %}
+    ignore_selected: {{ monitoring.netlink.ignore_selected }}
+    {%- if monitoring.netlink.interface_override and monitoring.netlink.interfaces is list %}
```

Comment:
i'd rather test is list and lenght of list > 0 instead of anothar param [interface_override]